Bottom: e80d5b85ada43a0c6d87988ba0ed140395d0bfc5
Top:    6bb933fd49224c099cfa1421310c068e6cf23583
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-15 01:58:45 +0000

Refresh of bga-wizard

---

diff --git a/src/hid/gtk/bga-wizard.c b/src/hid/gtk/bga-wizard.c
index c8518d1..e475192 100644
--- a/src/hid/gtk/bga-wizard.c
+++ b/src/hid/gtk/bga-wizard.c
@@ -142,7 +142,7 @@ static const char bga_wizard_help[] =
 int
 bga_wizard_action (int argc, char **argv, int px, int pz)
 {
-  if (!ghidgui || !ghidgui->ui_manager)
+  if (!ghidgui || !ghidgui->menu_bar)
     return 0;
 
   demo_assistant ();
